PWGJE/Tasks/nucleiWithFastJet.cxx

```diff
@@ -1113,83 +989,16 @@ struct nucleiWithFastJet {
         break;
       } // jet
 
-      if (track.mcParticle().pdgCode() == PDGProton) {
-        jetHist.fill(HIST("mcdJet/proton/pt/PtProton"), track.mcParticle().pt());
-        if (jetFlag) {
-          jetHist.fill(HIST("mcdJet/proton/pt/PtProton_jet"), track.mcParticle().pt());
-        } else {
-          jetHist.fill(HIST("mcdJet/proton/pt/PtProton_outJet"), track.mcParticle().pt());
-        }
+      if (mapPDGToValue(mcTrack.pdgCode())) {
```

Comment:
what does this line do? because it can be negative i am not sure how it works. Maybe instead you can have  if (mapPDGToValue(mcTrack.pdgCode()) != 0)

Reply:
Thanks @nzardosh for the suggestion, with the current version the if statement is false only for 0 and true for any other value. For clarity, I have impemented your suggestion.